src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/resource_allocation_manager_impl.cc

```diff
@@ -174,7 +174,7 @@ void ResourceAllocationManagerImpl::ProcessApplicationPolicyUpdate() {
       if (rc_extention) {
         rc_extention->UnsubscribeFromInteriorVehicleData(*module);
       }
-      SendOnRCStatusNotification();
+      SendOnRCStatusNotification(false);
```

Comment:
@LitvinenkoIra very unclear if you don't see function signature right now. Please use enum with 2 values against Boolean.